src/hooks/spaces/useSpaceById.ts

```diff
@@ -1,26 +1,27 @@
-import { COLLECTION_SPACES } from "settings";
+import { useMemo } from "react";
+
+import { COLLECTION_SPACES, DEFERRED } from "settings";
 
 import { LoadStatus } from "types/fire";
 import { SpaceId, SpaceWithId, WorldId } from "types/id";
 
-import { convertToFirestoreKey } from "utils/id";
-
-import { useRefiDocument } from "hooks/fire/useRefiDocument";
+import { useFireDocument } from "hooks/fire/useFireDocument";
 
-type UseSpaceById = (options: {
-  spaceId?: SpaceId;
-}) => LoadStatus & {
```

Comment:
This is the preferred signature. Just because now only `spaceId` is required, it doesn't mean in the future more options would not be needed.

Reply:
[8f7d0bc](https://github.com/sparkletown/sparkle/pull/2964/commits/8f7d0bcb22aad901b7d6456cb7b36d6787fe4ac1)